hera_cal/redcal.py

```diff
@@ -1409,8 +1409,32 @@ def redcal_iteration(hd, nInt_to_load=None, pol_mode='2pol', bl_error_tol=1.0, e
     return rv
 
 
+def _redcal_run_write_results(cal, hd, fistcal_filename, omnical_filename, omnivis_filename, 
+                              outdir, clobber=False, verbose=False, add_to_history=''):
+    '''Helper function for writing the results of redcal_run.'''
```

Comment:
Since it's a function with a leading underscore, it's not meant to be used outside this codebase (I honestly can't imagine a situation where it would be). If this becomes an issue in the future, I can promote it to a regular function.